subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -596,6 +597,27 @@ static int prepare_cb(struct lll_prepare_param *p)
 	return 0;
 }
 
+static int is_abort_cb(void *next, void *curr, lll_prepare_cb_t *resume_cb)
```

Comment:
`resume_cb` seems unused. Is that intentional?

Reply:
Added `ARG_UNUSED`

Reply:
`resume_cb` is a return parameter, and an `ASSERT` or any assignment is unnecessary when `-EAGAIN` is not the return value.

Reply:
> `resume_cb` is a return parameter, 

Ah, I had misunderstood the argument then. OK to leave as is :)